### PR TITLE
refactor(myscale):Set the default value of the myscale vector db in DifyConfig.

### LINKED ISSUE #6429: Set the default value of the myscale vector db in DifyConfig.
It's better to set the MyScale vector database's default in the Dify Config.

https://github.com/langgenius/dify/blob/c8f5dfcf17b0522789d0bff814c315bd2018a0a2/api/core/rag/datasource/vdb/myscale/myscale_vector.py#L162

### PR DESCRIPTION
**Pull request by ox01024**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [X] Please open an issue before creating a PR or link to an existing issue Close #6429 
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [X] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: api/configs/middleware/vdb/myscale_config.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class MyScaleConfig(BaseModel):
 
     MYSCALE_HOST: Optional[str] = Field(
```
Comment (laipz8200) [inline api/configs/middleware/vdb/myscale_config.py]: You can remove those `Optional` now.

Comment (ox01024) [reply api/configs/middleware/vdb/myscale_config.py]: Is it like this?

**Comment by laipz8200** [comment]
Please run `dev/reformat` to appease the lint gods

**Comment by ox01024** [comment]
My lousy memory, the great god of code review, you will definitely forgive me, right！

**Review by laipz8200 — APPROVED** [review]

**Pull request merged**